config/crd/bases/v1/datadoghq.com_datadogagentprofiles.yaml

```diff
@@ -0,0 +1,109 @@
+                                      type: object
+                                    requests:
+                                      additionalProperties:
+                                        anyOf:
+                                          - type: integer
+                                          - type: string
+                                        pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                                        x-kubernetes-int-or-string: true
+                                      description: 'Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
+                                      type: object
+                                  type: object
+                              type: object
+                            type: object
+                        type: object
+                      type: object
+                  type: object
+                dapAffinity:
+                  properties:
+                    dapNodeAffinity:
+                      items:
+                        description: A node selector requirement is a selector that contains values, a key, and an operator that relates the key and values.
+                        properties:
+                          key:
+                            description: The label key that the selector applies to.
+                            type: string
+                          operator:
+                            description: Represents a key's relationship to a set of values. Valid operators are In, NotIn, Exists, DoesNotExist. Gt, and Lt.
+                            type: string
+                          values:
+                            description: An array of string values. If the operator is In or NotIn, the values array must be non-empty. If the operator is Exists or DoesNotExist, the values array must be empty. If the operator is Gt or Lt, the values array must have a single element, which will be interpreted as an integer. This array is replaced during a strategic merge patch.
+                            items:
+                              type: string
+                            type: array
+                        required:
+                          - key
+                          - operator
+                        type: object
+                      type: array
+                  type: object
+              type: object
+            status:
+              description: DatadogAgentProfileStatus defines the observed state of DatadogAgentProfile
+              type: object
```

Comment:
Do we still want to include the resulting "active" DDA config in the status? The more I think about it, the more redundant it seems.

If we instead decide to include a simplified status, we can include: 
* `currentHash`: the stored hash of the resulting DaemonSet/Extendeddaemonset
* `state`: i.e. `Reconciled`, `Failed`, etc. 
* `conditions`: similar to the DDA [conditions](https://github.com/DataDog/datadog-operator/blob/58df38cd757b469a5ac4d324e0d0032cadf30b18/config/crd/bases/v1/datadoghq.com_datadogagents.yaml#L11247-L11293) field

Reply:
I'm not sure about including the "merged" DDA in the status either.

What you suggested sounds good to me :+1:

I'm not sure if we need to include the final version of the status in this PR. My suggestion is to leave it for later, or perhaps just include something simple like the `state` field you mentioned.